Module.php

```diff
@@ -8,12 +8,18 @@
 
 namespace ZfcUserImpersonate;
 
+use Zend\ModuleManager\Feature\ConfigProviderInterface;
 use Zend\Mvc\MvcEvent;
 use Zend\View\Model\ViewModel;
 use ZfcUserImpersonate\Module\AbstractModule;
 
-class Module extends AbstractModule
+class Module implements ConfigProviderInterface
 {
+    public function getConfig($env = null)
+    {
+        return include __DIR__ . '/config/module.config.php';
```

Comment:
When applying the `require_once` approach, as described below, `/../../` need to be added to the config file include.